### DISCUSSION TITLE
Welcome to Cocoon Discussions!

### DISCUSSION THREAD

**Discussion by ThatRendle** [VisualReCode/Cocoon#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eapr0427** [comment]
Hello @markrendle ,

I hope you are doing great. I am interested in using Cocoon to migrate my web forms app to a .NET core Blazor webserver app, using the one page at a time approach. I was wondering if there is a sample or guide available that could help me set this up.

I understand that this can be a complex process, and any guidance or resources you can provide would be greatly appreciated. I'm working on making the transition to Blazor as smooth and efficient as possible, and I think Cocoon could be a valuable tool in achieving this goal.

Thank you in advance for your help. I look forward to hearing from you.
